app/assets/javascripts/application.js

```diff
@@ -14,4 +14,8 @@
 //= require jquery_ujs
 //= require bootstrap-sprockets
 //= require turbolinks
-//= require_tree .
+//= require lib/knockout-3.4.0
+//= require lib/arbiter
+//= require bindings/scrollToBottom
+//= require bindings/toggler
+//= require view-models/chat
```

Comment:
Are you sure that we will need the chat in all pages, because `application.js` is load for all pages?